docs/azure_jumpstart_arcbox/_index.md

```diff
@@ -0,0 +1,159 @@
+
+## Azure Arc capabilities available in ArcBox
+
+### Azure Arc enabled Servers
+
+![ArcBox servers diagram](./servers.png)
+
+ArcBox includes three Azure Arc enabled server resources. These servers are hosted using nested virtualization in Azure. As part of the deployment, a Hyper-V host (ArcBox-Client) is deployed with three guest virtual machines. These machines, _ArcBoxWin_, _ArcBoxUbuntu_, and _ArcBoxSQL_ are connected as Azure Arc enabled servers via the ArcBox automation.
+
+### Azure Arc enabled Kubernetes
+
+![ArcBox Kubernetes diagram](./k8s.png)
+
+ArcBox deploys one single-node Rancher K3s cluster running on an Azure virtual machine. This cluster is then connected to Azure as an Azure Arc enabled Kubernetes resource (_ArcBox-K3s_).
+
+### Azure Arc enabled Data Services
+
+ArcBox deploys an AKS cluster, which is then used to deploy an Azure Arc enabled data services data controller.
+
+![ArcBox data services diagram](./dataservices.png)
+
+## Automation flow
+
+![Deployment flow diagram](./deploymentflow.png)
+
+ArcBox uses an advanced automation flow to deploy and configure all necessary resources with minimal user interaction. The above diagram provides a high level overview of the deployment flow. A high level summary of the deployment is:
+
+* User deploys the primary ARM template (azuredeploy.json). This template contains several nested templates that will run simultaneously.
+  * ClientVM ARM template - deploys the Client Windows VM
+  * AKS ARM template - deploys AKS cluster which will be used to run Azure Arc enabled data services
+  * Rancher K3s template - deploys a Linux VM which will have Rancher (K3s) installed on it and connected as an Azure Arc enabled Kubernetes cluster
+  * Storage account template - used for staging files in automation scripts
+  * Management artifacts template - deploys Log Analytics workspace and solutions and Azure Policy artifacts
+* User remotes into Client Windows VM, which automatically kicks off multiple scripts that:
+  * Deploy and configure three (3) nested virtual machines in Hyper-V
+    * Windows VM - onboarded as Azure Arc enabled Server
+    * Ubuntu VM - onboarded as Azure Arc enabled Server
+    * Windows VM running SQL Server - onboarded as Azure Arc enabled SQL Server
+  * Deploy and configure Azure Arc enabled data services on the AKS cluster including a data controller, a SQL MI instance, and a Postgres instance. After deployment, Azure Data Studio opens automatically with connection entries for each database instance. Data services deployed by the script are:
+    * Data controller
+    * SQL MI instance
+    * Postgres instance
+
+## Deployment Option 1: Azure Portal
+
+* Click the [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fazure_arc%2Farcbox%2Fazure_jumpstart_arcbox%2Fazuredeploy.json) button and enter values for the the ARM template parameters.
+
```

Comment:
Add a screenshot of the Azure portal with the parameters view. I also recommend adding the "Deployment in progress" screen and "deployment completed" screen.